response/ta_send_mam.h

```diff
@@ -27,6 +27,8 @@ typedef struct send_mam_res_s {
   char bundle_hash[NUM_TRYTES_HASH + 1];
   /** ascii string channel id */
   char channel_id[NUM_TRYTES_HASH + 1];
+  /** channel ordinal */
+  int32_t channel_ord;
```

Comment:
You shall explain the motivation to extend the type of channel ordinal.

Reply:
?

Reply:
I have added in the commit message. Let me add it in comment as well